results/Pseudomonas_analysis_sample_lvl_sim_svcca_0.05.svg

```diff
@@ -0,0 +1,1962 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
Might be good for these files to have the same naming scheme as the original experiments (something like `Pa_sample_lvl...`). Up to you though.

Reply:
Good catch...the "Pa_" is from an earlier version of code.  I have changed the filenames to be more consistent